cmd/nomos/util/write.go

```diff
@@ -31,8 +31,8 @@ const (
 	NotConfiguredMsg = "NOT CONFIGURED"
 	// UnknownMsg indicates that a field's value is unknown or unavailable.
 	UnknownMsg = "UNKNOWN"
-	// OSSMsg indicates OSS installation
-	OSSMsg = "OSS Installation"
+	// StandaloneMsg indicates standalone OSS installation
+	StandaloneMsg = "Standalone Installation"
```

Comment:
Here will be the new output:
```
nomos version --contexts=$(kubectl config current-context)
CURRENT   CLUSTER_CONTEXT_NAME                                    COMPONENT           VERSION
                                                                  <nomos CLI>         UNKNOWN
*         gke_nanyu-gke-dev_us-central1-c_cs-standalone-regular   config-management   Standalone Installation
```

The COMPONENT column shows `config-management`, and the VERSION column shows `Standalone Installation`.

We'll need to change the COMPONENT column to `config-sync`, which distinguish the standalone Config Sync from the ACM version of Config Sync. For the VERSION column, it would be good to extract the image tag from the reconciler-manager container. WDYT? @sdowell

Reply:
That's a good suggestion, SGTM

Reply:
> it would be good to extract the image tag from the reconciler-manager container 

since nomos version would normally query from all clusters and project in kubeconfig, will it able to access the image tag from different clusters and projects?

Reply:
As long as the clusters are accessible, it should be able to get the reconciler-manager Deployment and extract the image tag.

Reply:
I made a prototype of this in my local, some example output: 
``` 
CURRENT   CLUSTER_CONTEXT_NAME                                       COMPONENT           VERSION
                                                                     <nomos CLI>         v1.14.3-rc.2-5-g5e27395-dirty
          gke_victortjhia-gke-dev_us-central1-c_operator-cluster-1   config-management   v1.14.1-rc.3
          gke_victortjhia-gke-dev_us-central1-c_oss-cluster-1        config-sync         latest
*         gke_victortjhia-gke-dev_us-central1-c_oss-cluster-v114     config-sync         v1.14.3-rc.2-5-g5e27395
          gke_victortjhia-gke-dev_us-east4-c_nat-test-cluster                            <error: Get "https://172.16.0.2/api?timeout=3s": net/http: request canceled while waiting for connection (Client.Timeout exceeded while awaiting headers)>

```

(the one showing "latest" as version was because I was following an out of date installation instruction for config sync)

Reply:
The output looks good to me.